nautobot/extras/plugins/views.py

```diff
@@ -86,10 +86,8 @@ class InstalledAppsView(GenericView):
     """
 
     table = InstalledAppsTable
-    breadcrumbs = Breadcrumbs(
-        items={"generic": [ViewNameBreadcrumbItem(view_name="apps:apps_list", label="Installed Apps")]}
-    )
-    view_titles = Titles(titles={"generic": "Installed Apps"})
+    breadcrumbs = Breadcrumbs(items={"*": [ViewNameBreadcrumbItem(view_name="apps:apps_list", label="Installed Apps")]})
+    view_titles = Titles(titles={"*": "Installed Apps"})
```

Comment:
Is there some breaking change to the ui-components code that necessitates changing these from `generic` to `*`, or just a matter of stylistic preference here?